test/executor/test_insert_executor.py

```diff
@@ -0,0 +1,72 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+from src.parser.parser import Parser
+from src.optimizer.statement_to_opr_convertor import StatementToPlanConvertor
+from src.planner.insert_plan import InsertPlan
+from src.planner.create_plan import CreatePlan
+from src.catalog.models.df_column import DataFrameColumn
+from src.catalog.column_type import ColumnType
+from src.parser.table_ref import TableRef, TableInfo
+from src.executor.insert_executor import InsertExecutor
+from src.executor.create_executor import CreateExecutor
+from src.storage.dataframe import load_dataframe
+
+
+class InsertExecutorTest(unittest.TestCase):
```

Comment:
This looks like an integration test. 

We should keep integration tests from unit tests.

Reply:
Moved to a separate integration test directory